frontend/openchat-client/src/openchat.ts

```diff
@@ -2382,56 +2386,25 @@ export class OpenChat extends OpenChatAgentWorker {
-                const resp = await this.sendRequest({
-                    kind: "getGroupDetails",
-                    chatId: clientChat.id,
-                    timestamp: clientChat.lastUpdated,
-                });
-                if (resp !== "failure") {
-                    chatStateStore.setProp(clientChat.id, "detailsLoaded", true);
-                    chatStateStore.setProp(clientChat.id, "members", resp.members);
-                    chatStateStore.setProp(clientChat.id, "blockedUsers", resp.blockedUsers);
-                    chatStateStore.setProp(clientChat.id, "invitedUsers", resp.invitedUsers);
-                    chatStateStore.setProp(clientChat.id, "pinnedMessages", resp.pinnedMessages);
-                    chatStateStore.setProp(clientChat.id, "rules", resp.rules);
-                    chatStateStore.setProp(clientChat.id, "lastUpdated", resp.timestamp);
-                }
-                await this.updateUserStore(clientChat.id, []);
-            } else {
-                await this.updateChatDetails(clientChat);
-            }
-        }
-    }
-
-    private async updateChatDetails(clientChat: ChatSummary): Promise<void> {
-        if (clientChat.kind === "group_chat" || clientChat.kind === "channel") {
-            const timestamp = chatStateStore.getProp(clientChat.id, "lastUpdated");
-            if (timestamp !== undefined && timestamp < clientChat.lastUpdated) {
-                const gd = await this.sendRequest({
-                    kind: "getGroupDetailsUpdates",
-                    chatId: clientChat.id,
-                    previous: {
-                        members: chatStateStore.getProp(clientChat.id, "members"),
-                        blockedUsers: chatStateStore.getProp(clientChat.id, "blockedUsers"),
-                        invitedUsers: chatStateStore.getProp(clientChat.id, "invitedUsers"),
-                        pinnedMessages: chatStateStore.getProp(clientChat.id, "pinnedMessages"),
-                        timestamp: chatStateStore.getProp(clientChat.id, "lastUpdated"),
-                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                        rules: chatStateStore.getProp(clientChat.id, "rules")!,
-                    },
-                });
-                chatStateStore.setProp(clientChat.id, "members", gd.members);
-                chatStateStore.setProp(clientChat.id, "blockedUsers", gd.blockedUsers);
-                chatStateStore.setProp(clientChat.id, "invitedUsers", gd.invitedUsers);
-                chatStateStore.setProp(clientChat.id, "pinnedMessages", gd.pinnedMessages);
-                chatStateStore.setProp(clientChat.id, "rules", gd.rules);
-                chatStateStore.setProp(clientChat.id, "lastUpdated", gd.timestamp);
-                await this.updateUserStore(clientChat.id, []);
+        if (serverChat.kind === "group_chat" || serverChat.kind === "channel") {
```

Comment:
can the `detailsLoaded` be removed from the `ChatSpecificState` entirely now as well?